code/datums/mutations/actions.dm

```diff
@@ -216,7 +216,7 @@
 
 /datum/mutation/overload
 	name = "Overload"
-	desc = "Allows an Ethereal to overload their skin to cause a bright flash."
+	desc = "Allows an Ethereal to overload their skin, causing a bright flash."
```

Comment:
The original description is grammatically correct: "Allows an Ethereal to overload their skin [in order] to cause a bright flash.". It's a purpose clause.